packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/utils.spec.ts

```diff
@@ -21,59 +25,83 @@ const tableColumnConfig: WfoTableColumnConfig<TestObject> = {
+            expect(result).toBe(SortOrder.DESC);
+        });
+        it('returns SortOrder.ASC if the currentSortOrder is SortOrder.DESC', () => {
+            const currentSortOrder = SortOrder.DESC;
+            const result = getUpdatedSortOrder(currentSortOrder);
+            expect(result).toBe(SortOrder.ASC);
+        });
+        it('returns SortOrder.ASC if the currentSortOrder is undefined', () => {
+            const currentSortOrder = undefined;
+            const result = getUpdatedSortOrder(currentSortOrder);
+            expect(result).toBe(SortOrder.ASC);
+        });
+    });
+
+    describe('mapSortableAndFilterableValuesToTableColumnConfig()', () => {
+        it('sets the sortable and filterable properties for the columnConfig object to true when the colum name is specified in the list', () => {
+            // Given
+            const sortableFieldNames = ['name', 'age', 'nonExistingFieldName'];
+            const filterableFieldNames = [
+                'name',
+                'age',
+                'nonExistingFieldName',
+            ];
 
-        // When
-        const result =
-            mapSortableAndFilterableValuesToTableColumnConfig<TestObject>(
-                tableColumnConfig,
-                sortableFieldNames,
-                filterableFieldNames,
-            );
+            // When
+            const result =
+                mapSortableAndFilterableValuesToTableColumnConfig<TestObject>(
+                    tableColumnConfig,
+                    sortableFieldNames,
+                    filterableFieldNames,
+                );
 
-        // Then
-        if (
-            result.name?.columnType === ColumnType.DATA &&
-            result.age?.columnType === ColumnType.DATA
-        ) {
-            expect(result.name.isSortable).toEqual(true);
-            expect(result.name.isFilterable).toEqual(true);
-            expect(result.age.isSortable).toEqual(true);
-            expect(result.age.isFilterable).toEqual(true);
-        } else {
-            // Preventing silently skipping above expects
-            throw Error('Some of the fields are not data fields');
-        }
-    });
-    it('sets the sortable and filterable properties for the columnConfig object to false when the colum name is not specified in the list', () => {
-        // Given
-        const sortableFieldNames = ['nonExistingFieldName'];
-        const filterableFieldNames: string[] = [];
+            // Then
+            if (
+                result.name?.columnType === ColumnType.DATA &&
+                result.age?.columnType === ColumnType.DATA
+            ) {
+                expect(result.name.isSortable).toEqual(true);
+                expect(result.name.isFilterable).toEqual(true);
+                expect(result.age.isSortable).toEqual(true);
+                expect(result.age.isFilterable).toEqual(true);
+            } else {
+                // Preventing silently skipping above expects
```

Comment:
In what case would you end up in this branch? Don't you always get the same results? I see this was already there but maybe we can remove it

Reply:
TypeScript is unable to determine the column type of the `name` and `age` columns, the check is needed to avoid TSC errors. We need the else branch to make the unit test fail when skipped